src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -99,6 +112,30 @@ export default class CancerStudyTreeData {
         // sort by name
         cancerTypes = CancerStudyTreeData.sortNodes(cancerTypes);
 
+        //map public virtual study to cancer study
+        const _publicVirtualStudies = publicVirtualStudies
+            .map(publicVirtualStudy => {
+                // TODO: temp fix for when virtual study data is not of expeceted format

```

Comment:
will this be added or is it a separate ticket?

Reply:
nea, let's just clean it. https://github.com/cBioPortal/cbioportal-frontend/pull/4923/commits/2ac225df6e76712b2da315aeb08444226707b069